content/en/synthetics/private_locations/_index.md

```diff
@@ -465,7 +485,70 @@ Because Datadog already integrates with Kubernetes and AWS, it is ready-made to
+
+   Logging Verbosity
+   : Specifies the verbosity of the console and file logging for the Synthetics Private Location Worker. Options for verbosity when performing an unattended `msiexec` installation include `-v` for error, `-vv` for information, and `-vvv` for debug.
+
+1. Click **Next** and **Install** to start the installation process. 
+   
+   {{< img src="synthetics/private_locations/settings.png" alt="Configuration options in the MSI installer" style="width:80%;" >}}
+   
+Once the process is complete, click **Finish** on the installation completion page.
+
+<div class="alert alert-warning">If you have entered your JSON configuration, the Windows Service starts running using that configuration. If you did not enter your configuration, run <code>synthetics-pl-worker.exe --config=<PathToYourConfiguration></code> from a command prompt or use the <code>start menu</code> shortcut to start the Synthetics Private Location Worker.</div>
+
+To install the Synthetics Private Location Worker on the command line, use `msiexec` and include parameters as needed. 
+
+| Optional Parameter | Definition | Value | Default Value | Type |
+|---|---|---|---|---|
+| APPLYDEFAULTFIREWALLRULES | Applies firewall rules needed for the program. | 1 | N/A | 0: Disabled<br>1: Enabled |
+| APPLYFIREWALLDEFAULTBLOCKRULES | Blocks reserved IP addresses for each browser you have installed (Chrome, Edge, and Firefox). Blocking loopback connections is not possible in Windows Firewall. | 0 | N/A | 0: Disabled<br>1: Enabled |
+| LOGGING_ENABLED | When enabled, this configures file logging. These logs are stored in the installation directory under the logs folder. | 0 | `--enableFileLogging` | 0: Disabled<br>1: Enabled |
+| LOGGING_VERBOSITY | Configures the logging verbosity for the program. This affects console and file logs. | This affects console and file logs. | `-vvv` | `-v`: Error<br>`-vv`: Info<br>`-vvv`: Debug |
+| LOGGING_MAXDAYS | Number of days to keep file logs on the system before deleting them. Can be any number when running an unattended installation. | 7 | `--logFileMaxDays` | Integer |
+| WORKERCONFIG_FILEPATH | This should be changed to the path to your Synthetics Private Location Worker JSON configuration file. Wrap this path in quotes if your path contains spaces. | <None> | `--config` | String |
+
+For example, this command applies the default firewall rules needed by the program to configure the Windows service: 
+
+```shell
+msiexec /i datadog-synthetics-worker-<version>.amd64.msi /quiet /qn APPLYDEFAULTFIREWALLRULES=1 APPLYFIREWALLDEFAULTBLOCKRULES=1 LOGGING_ENABLED=1 LOGGING_VERBOSITY=”-vv” LOGGING_MAXDAYS=7 WORKERCONFIG_FILEPATH=C:\ProgramData\Datadog-Synthetics\worker-config.json
+```
+
+
+To uninstall the Synthetics Private Location Worker on the command line, use `msiexec`. For example, you can run `msiexec /uninstall datadog-synthetics-worker-<version>.amd64.msi /passive`.
```

Comment:
can we make this command match the same format as the one right above? (i would just do this as a suggestion, but using three backticks seems to really mess with github suggestions)